config/src/test/kotlin/org/springframework/security/config/annotation/web/RememberMeDslTests.kt

```diff
@@ -14,32 +14,26 @@
  * limitations under the License.
  */
 
-package org.springframework.security.config.web.servlet
+package org.springframework.security.config.annotation.web
 
-import io.mockk.Called
```

Comment:
Nice catch finding these unused imports @nor-ek!
Since this change is not related to resolving package tangles, please move it to a separate commit.
This will ensure the commit message accurately describes the changes.

Reply:
@eleftherias please check it out. I've noticed more changes like that and packed them into `Import cleanup` commit. 

I've also cleaned up titles and messages.